chain/jsonrpc/src/lib.rs

```diff
@@ -1310,6 +1347,18 @@ fn status_handler(
     response.boxed()
 }
 
+fn debugz_handler(
+    handler: web::Data<JsonRpcHandler>,
+) -> impl Future<Output = Result<HttpResponse, HttpError>> {
+    let response = async move {
+        match handler.debugz().await {
+            Ok(value) => Ok(HttpResponse::Ok().json(&value)),
+            Err(_) => Ok(HttpResponse::MethodNotAllowed().finish()),
+        }
+    };
+    response.boxed()
+}
```

Comment:
I wonder if this can be just

```suggestion
async fn debugz_handler(
    handler: web::Data<JsonRpcHandler>,
) -> Result<HttpResponse, HttpError> {
        match handler.debugz().await {
            Ok(value) => Ok(HttpResponse::Ok().json(&value)),
            Err(_) => Ok(HttpResponse::MethodNotAllowed().finish()),
        }
}
```

?

Can't check immediatelly -- waiting for nearcore to compile on my trusty old laptop :) Already miss my 12 core machine :(